Updating Useful Terms (#4831)
cloudflare/cloudflare-docs: content/cloudflare-one/connections/connect-apps/install-and-setup/tunnel-useful-terms.md

## Connector

> The connector, simply called `cloudflared`, establishes connectivity from your origin server to the Cloudflare global network. Each instance of the connector will be given a unique connector id which can be surfaced by running `cloudflared tunnel info <NAME or UUID>` or visiting the Zero Trust dashboard. Our connector offers high-availability by design, creating four long-lived connections to two distinct data centers within Cloudflare’s global network. This means that whether an individual connection, server, or data center goes down, your origin remains available.

Reviewer: ```suggestion
The connector, referred to as `cloudflared`, establishes connectivity from your origin server to the Cloudflare global network. Our connector offers high availability by creating four long-lived connections to two distinct data centers within Cloudflare’s global network. Therefore, regardless of whether an individual connection, server, or data center goes down, your origin remains available.
```

  
## Replica

> You can create and configure a tunnel once and run that tunnel through multiple, unique instances of the connector, `cloudflared`. These instances are known as connectors, or replicas. DNS records and Cloudflare Load Balancers will still point to the tunnel and its DNS Record (UUID.cfargotunnel.com), while that tunnel sends traffic to the multiple instances of cloudflared that run through it. Today, there is no guarentee about which connector will be chosen. Replicas are often deployed to provide tunnels with high availability in the event a given host running cloudflared is interupted or taken offline. 

Reviewer 2: s/guarentee/guarantee

Author: Good catch 😅

> You can create and configure a tunnel once and run that tunnel through multiple, unique instances of the connector, `cloudflared`. These instances are known as connectors, or replicas. DNS records and Cloudflare Load Balancers will still point to the tunnel and its DNS Record (UUID.cfargotunnel.com), while that tunnel sends traffic to the multiple instances of cloudflared that run through it. Today, there is no guarentee about which connector will be chosen. Replicas are often deployed to provide tunnels with high availability in the event a given host running cloudflared is interupted or taken offline. 

Reviewer: ```suggestion
You can create and configure a tunnel once and run that tunnel through multiple, unique instances of the connector, `cloudflared`. These instances are known as replicas. DNS records and Cloudflare Load Balancers will still point to the tunnel and its DNS Record (`UUID.cfargotunnel.com`), while that tunnel sends traffic to the multiple instances of `cloudflared` that run through it. Today, there is no guarantee about which replica will be chosen. Replicas are often deployed to provide tunnels with high availability in the event a given host running `cloudflared` is interrupted or taken offline. 
```
